File: EJER-ARREGLOS.java
import java.util.*;

class ventas{
    static String[] departamentos = {"ropa", "deportes", "jugueteria"};
    static String[] meses = {"enero", "febrero", "marzo", "abril", "mayo", "junio", 
                             "julio", "agosto", "septiembre", "octubre", "noviembre", "diciembre"};
    static int[][] ventas = new int[12][3];

    public static void insertarVenta(String mes, String departamento, int cantidad) {
        mes = mes.toLowerCase().replace(" ", "");
        departamento = departamento.toLowerCase().replace(" ", "");
        int indiceMes = Arrays.asList(meses).indexOf(mes);
        int indiceDepto = Arrays.asList(departamentos).indexOf(departamento);

        if (indiceMes != -1 && indiceDepto != -1) {
            ventas[indiceMes][indiceDepto] = cantidad;
            System.out.println("Venta insertada con éxito");
        } else {
            System.out.println("Mes o departamento inválido");
        }
    }

    public static String buscarVenta(String mes, String departamento) {
        mes = mes.toLowerCase().replace(" ", "");
        departamento = departamento.toLowerCase().replace(" ", "");
        int indiceMes = Arrays.asList(meses).indexOf(mes);
        int indiceDepto = Arrays.asList(departamentos).indexOf(departamento);

        if (indiceMes != -1 && indiceDepto != -1) {
            return "La venta del mes " + mes + " para el departamento " + departamento +
                   " es: " + ventas[indiceMes][indiceDepto];
        } else {
            return "Mes o departamento inválido";
        }
    }

    public static void eliminarVenta(String mes, String departamento) {
        mes = mes.toLowerCase().replace(" ", "");
        departamento = departamento.toLowerCase().replace(" ", "");
        int indiceMes = Arrays.asList(meses).indexOf(mes);
        int indiceDepto = Arrays.asList(departamentos).indexOf(departamento);

        if (indiceMes != -1 && indiceDepto != -1) {
            ventas[indiceMes][indiceDepto] = 0;
            System.out.println("Venta eliminada con éxito");
        } else {
            System.out.println("Mes o departamento inválido");
        }
    }

    public static void main(String[] args) {
        insertarVenta("Enero", "Ropa", 1500);
        System.out.println(buscarVenta("Enero", "Ropa"));
        insertarVenta("Febrero", "Deportes", 2000);
        insertarVenta("Febrero", "Ropa", 2000);
        insertarVenta("Febrero", "Jugueteria", 2000);
        insertarVenta("Marzo", "Jugueteria", 3000);
        insertarVenta("Abril", "Ropa", 2500);
        System.out.println("\nMatriz de ventas:");
        for (int[] filaVentas : ventas) {
            System.out.println(Arrays.toString(filaVentas));
        }
    }
}

File: EJER - ARREGLOS.py
departamentos = ["ropa", "deportes", "jugueteria"]
meses = ["enero", "febrero", "marzo", "abril", "mayo", "junio", "julio", "agosto", "septiembre", "octubre", "noviembre", "diciembre"]
# Crearemos el arreglo bidimensional
ventas = [
    [0 for i in range(3)] for i in range(12)
]

def insertar_venta(mes, departamento, cantidad):
    mes = mes.lower().replace(" ", "")
    departamento = departamento.lower().replace(" ", "")
    if mes in meses  and departamento in departamentos:
        ventas[meses.index(mes)][departamentos.index(departamento)] = cantidad
        print(f"venta inserada con exito")
    else:
        print("Mes o departamento inválido")

def buscar_venta(mes, departamento):
    mes = mes.lower().replace(" ", "")
    departamento = departamento.lower().replace(" ", "")
    if mes in meses  and departamento in departamentos:
        return f"la venta del mes {mes} para el departamento {departamento} es: {ventas[meses.index(mes)][departamentos.index(departamento)]}"
    else:
        print("Mes o departamento inválido")
        
def eliminar_venta(mes, departamento):
    mes = mes.lower().replace(" ", "")
    departamento = departamento.lower().replace(" ", "")
    if mes in meses  and departamento in departamentos:
        ventas[meses.index(mes)][departamentos.index(departamento)] = 0
        print("Venta eliminada con exito")
    else:
        print("Mes o departamento inválido")
        
# pruebas:
# insertar_venta("Enero", "Ropa", 1500)
# print(buscar_venta("Enero", "Ropa"))
# print(ventas)